src/library/Main.java

```diff
@@ -54,166 +54,167 @@ private static String Get_menu() {
 
 	public static void main(String[] args) {		
 		try {			
-			IN = new Scanner(System.in);
-			LIB = Library.GeTiNsTaNcE();
-			CAL = Calendar.gEtInStAnCe();
-			SDF = new SimpleDateFormat("dd/MM/yyyy");
+			input = new Scanner(System.in);
+			library = Library.getInstance();
+			calender = Calendar.getInstance();
+			date = new SimpleDateFormat("dd/MM/yyyy");
 	
-			for (Member m : LIB.lIsT_MeMbErS()) {
-				output(m);
+			for (Member member : library.listMembers()) {
+				output(member);
 			}
 			output(" ");
-			for (Book b : LIB.lIsT_BoOkS()) {
-				output(b);
+			for (Book book : library.listBooks()) {
+				output(book);
 			}
 						
-			MENU = Get_menu();
+			menu = getMenu();
 			
-			boolean e = false;
+			boolean exit = false;
 			
-			while (!e) {
+			while (!exit) {
 				
-				output("\n" + SDF.format(CAL.gEt_DaTe()));
-				String c = input(MENU);
+				output("\n" + date.format(calender.getDate()));

```

Comment:
Not 100% sure on this but do you think this conflicts with #9 in the styleguide?
> The argument list of any method calls MUST NOT contain any other method calls